lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -121,6 +123,17 @@ export type ClientCredentials = {
     };
 };
 
+/**
+ * AzureCloudInstance specific options
+ *
+ * - azureCloudInstance             - string enum providing short notation for soverign and public cloud authorities
+ * - tenant                         - provision to provide the tenant info
+ */
+export type AzureAuthOptions = {
+    azureCloudInstance?: AzureCloudInstance;
+    tenant: string,
```

Comment:
I would propose teh following:

```suggestion
    cloudInstance?: AzureCloudInstance;
    tenantId: string,
```

Reply:
If we plan to add PPE and other clouds, then yes, may be we can make it `cloudInstance`. Let me check with .Net folks if we can expand the definition to give them a heads up (for any future compat asks that can come to them).

Reply:
Keeping it `azureCloudInstance` to comply with MSAL compat